src/main/kotlin/Network.kt

```diff
@@ -14,11 +14,14 @@ class Network(val capacity : Int) {
         }
     }
 
-    fun analyzeNetwork() {
+    fun analyzeNetwork() : String {
         val overCapacity = this.overCapacity()
         for (car in this.listOfCarsDriving) {
             car.delayed = overCapacity
+            val isDelayed = car.delayed
+            return "$isDelayed"
```

Comment:
When you return inside a loop, the loop stops immediately. I don't think this is what you intended. :smiley:  I actually see no reason to change this function.